packages/sources/src/addSourceWizard/hardcodedComponents/openshift/costManagement.js

```diff
@@ -0,0 +1,222 @@
+import { TextContent, Text, TextVariants, TextListItem, TextList, Button, ClipboardCopy, Popover, ButtonVariant } from '@patternfly/react-core';
+import { global_spacer_md, global_spacer_xl } from '@patternfly/react-tokens';
+import { QuestionCircleIcon } from '@patternfly/react-icons';
+import React from 'react';
+
+export const PrerequisiteDescription = () => (
+    <TextContent key='description'>
+        <Text component={TextVariants.p}>
+            Before you begin, install the following prerequisites if not present:
+        </Text>
+    </TextContent>
+);
+
+export const PrerequisiteOCPText = () => (
+    <TextContent key='prerequisite-ocp'>
+        <Text component={TextVariants.p}>
+            On your OpenShift cluster, install:
+        </Text>
+    </TextContent>
+);
+
+export const PrerequisiteSystemText = () => (
+    <TextContent key='prerequisite-system'>
+        <Text component={TextVariants.p}>
+            On a system with network access to your OpenShift cluster, install:
+        </Text>
+    </TextContent>
+);
+
+export const PrerequisiteOCPList = () => (
+    <TextContent className='list-align-top'>
+        <TextList component='ul'>
+            <TextListItem component='li'>
+                <Text
+                    component={TextVariants.a}
+                    target="_blank"
+                    href="https://docs.openshift.com/container-platform/4.2/getting_started/install_openshift.html"
+                >
+                    OpenShift Container Platform 4.2 or newer
+                </Text>
+            </TextListItem>
+            <TextListItem component='li'>
+                <Text
+                    component={TextVariants.a}
+                    target="_blank"
+                    href="https://docs.openshift.com/container-platform/4.2/metering/metering-installing-metering.html"
+                >
+                        Operator Metering
+                </Text>
+            </TextListItem>
+        </TextList>
+    </TextContent>
+);
+
+export const PrerequisiteSystemList = () => (
+    <TextContent className='list-align-top'>
+        <TextList component='ul'>
+            <TextListItem component='li'>
+                <Text component={TextVariants.a} target="_blank" href="htps://access.redhat.com/products/red-hat-insights#getstarted" >Red Hat Insights Client</Text>
+            </TextListItem>
+            <TextListItem component='li'>
+                <Text
+                    component={TextVariants.a}
```

Comment:
You should also add `rel="noopener noreferrer"` to all `a` elements.

Reply:
@boaz0 Can you please fix this? Thanks!

Reply:
Sure!

Reply:
Done!